hardware/esp8266com/esp8266/libraries/ESP8266WiFi/src/WiFiClient.cpp

```diff
@@ -40,10 +40,15 @@ extern "C"
 #include "include/ClientContext.h"
 #include "c_types.h"
 
+
+#define NO_PORT_BIND
```

Comment:
What is the effect of NO_PORT_BIND? If we use it always for any reason, then just delete the whole #ifndef NO_PORT_BIND code path.